features/parse-html-unsafe.yml

```
@@ -0,0 +1,7 @@
+name: Unsafe HTML parsing methods
+description: The `Document.parseHTMLUnsafe()` static method and `setHTMLUnsafe()` method of `Element` and `ShadowRoot` TODO.
```

Just leaving an idea here that if santization or safety is part of the framing for this, then we should just go ahead and give readers a handle on that directly. The last sentence here, for instance:

```suggestion
description: The `Document.parseHTMLUnsafe()` static method parses HTML into a DOM tree, while the `setHTMLUnsafe()` method of `Element` and `ShadowRoot` parses and inserts HTML into an existing tree. No sanitization applies to these methods, so never call them with user-provided HTML strings.
```